C/CoolProp/build_tarballs.jl

```diff
@@ -29,7 +32,7 @@ install_license $WORKSPACE/srcdir/CoolProp*/LICENSE
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
-platforms = supported_platforms()
+platforms = supported_platforms(; experimental=true)
```

Comment:
```suggestion
platforms = supported_platforms()
```

Reply:
Was wondering about that, looks like we both made the change :)